tools/transformer/src/transform.js

```diff
@@ -21,9 +21,9 @@ if (apiFileLocation) {
   throw new Error("Missing location. Please set OPENAPI_FILE env variable");
 }
 
+doc = applyModelNameTransformations(doc, "ApiAtlas", "View");
+
 doc = applyOneOfTransformations(doc);
 doc = applyAllOfTransformations(doc);
```

Comment:
applyAllOfTransformations  - should happen first before  applyOneOfTransformations

Reply:
Yup missed that - will update

Reply:
Actually on this - there are some cases where we are doing oneOf transformations to models that would qualify for the allOf transformation, i.e. `ApiAtlasCloudProviderAccessRoleView` (but more importantly), `ApiAtlasDedicatedHardwareSpecView`. So I think it would be best to keep the transformations in this order.

Reply:
Sounds good. We can handle oneOf propery transformation in separate PR. Curious what sdk would look like we we also apply that across the board